content/docs/references/cli/_index.md

```diff
@@ -782,3 +783,58 @@ Encryption rekey status with name of the rekey object
 # dellctl encryption rekey-status myrekey
  INFO Status of rekey request myrekey = completed
 ```
+
+
+### dellctl images
+
+Prints a Table of the container images Driver needs
+
+**NOTE.**: dellctl images Currently supports csi-vxflexos driver only.
+
```

Comment:
Similar sentence rephrasing is needed here as well "List the container images used by driver."
Thanks.

Reply:
done